providers/evercrypt/fst/EverCrypt.HMAC.fst

```diff
@@ -3,6 +3,55 @@ module EverCrypt.HMAC
 
 open Hacl.HMAC
 
+// EverCrypt.Hash.Incremental.hash_256 is marked as private, so it is
+// inaccessible from here. Furthermore, the EverCrypt.Hash.Incremental module
+// does not have an interface, meaning that we can't even friend it! Writing an
+// interface for EverCrypt.Hash.Incremental is possible, but doesn't make much
+// sense: instantiations of the functor offer, by construction, an abstract
+// interface. Fortunately, we can use tactics to stealthily gain access to a
+// private definition that F* normally won't let us access.
+let super_hack () =
```

Comment:
It would be nice to mention this hack in one of the tutorials (F* or Low*) :)

Reply:
But then too many people might become aware of it, and it might get removed... tricky!!